packages/table/src/TableRow.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
```

Comment:
Try to use direct imports if you can so we can benefit from tree shaking. eg: `import includes from 'lodash/includes'`